client/src/paths/Trash.tsx

```diff
@@ -136,20 +115,15 @@ export function Trash() {
   );
 
   return (
-    <>
+    <Box
+      data-test="Activities"
+      width="100%"
```

Comment:
Set this width to
`width={{ base: "100%", md: "calc(100% - 40px)" }}`
as discussed in Activities.